Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+    status = outputs.get('status')
+    if status == 'Completed':
+        # # action was completed
+        if global_integration_context := demisto.getIntegrationContext():
+            del global_integration_context['command_id']
+            demisto.setIntegrationContext(global_integration_context)
+        return results_func(client, args)
+    elif status == 'Error':
+        if global_integration_context := demisto.getIntegrationContext():
+            del global_integration_context['command_id']
+            demisto.setIntegrationContext(global_integration_context)
+
+        return command_result
+    else:
+        # in case of In progress
+        polling_args = {
+            'interval_in_seconds': interval_in_secs,
+            'polling': True,
+            **args,
+        }
+        scheduled_command = ScheduledCommand(
+            command=cmd,
+            next_run_in_seconds=interval_in_secs,
+            args=polling_args,
+            timeout_in_seconds=timeout_in_seconds
+        )
+
+        # result with scheduled_command only - no update to the war room
+        return CommandResults(
+            scheduled_command=scheduled_command,
+            ignore_auto_extract=True
+        )
+
+
+def file_scheduled_polling_command(client: Client, args: Dict[str, Any]):
+    """
+    File Scheduled Polling file command
+    Returns:
+        return polling CommandResults
+    """
+    return run_polling_command(client, args, 'file', check_sandbox_status, get_sandbox_verdict)
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """
+    main function, parses params and runs command functions
+    """
+    try:
+        params = demisto.params()
+        args = demisto.args()
+        command = demisto.command()
+
+        # OAuth parameters
+        server_url = params.get('url', '')
+        client_id = params.get('credentials', {}).get('identifier', '')
+        client_secret = params.get('credentials', {}).get('password', '')
+        verify_certificate = params.get('insecure', False)
+        proxy = params.get('proxy', False)
+
+        # Fetches Incident Parameters
+        first_fetch_time = params.get('first_fetch', '3 days').strip()
+        fetch_limit = arg_to_number(params.get('max_fetch', 50))
+        fetch_incident_event = params.get('isIncidentsEvent', False)
+        fetch_comments = params.get('isIncidentComment', False)
+        fetch_status = argToList(params.get('fetch_status', 'New'))
+        fetch_priority = argToList(params.get('fetch_priority', 'High,Medium'))
+
+        client = Client(base_url=server_url, verify=verify_certificate, proxy=proxy, client_id=client_id,
+                        client_secret=client_secret, first_fetch=first_fetch_time, fetch_limit=fetch_limit,
+                        is_incident_event=fetch_incident_event, is_fetch_comment=fetch_comments,
+                        fetch_status=fetch_status, fetch_priority=fetch_priority)
+
+        client.get_access_token_or_login()
```

Comment:
can we move this into the `Client`'s constructor?

Reply:
Done. kindly review.